sandbox.sh

```diff
@@ -22,7 +22,7 @@ tezos-client() {
 }
 
 ligo() {
-  docker run --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:0.28.0 "$@"
+  docker run --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:0.37.0 "$@"
```

Comment:
Shall we?
I cannot make it work on my debian with the 0.28 version 🤷

Reply:
Likely a good choice